#Массив значений короткого замыкания на ключах системы
uint8[16] status